sqlitecluster/SQLiteNode.h

```diff
@@ -349,6 +352,11 @@ class SQLiteNode : public STCPManager {
     // Remove. See: https://github.com/Expensify/Expensify/issues/208449
     atomic<int> _priority;
 
+    // These three variables are used to coordinate the startup of replication threads to guarantee each thread starts before we attempt to start the next one.
+    mutex _replicateStartMutex;
+    condition_variable _replicateStartCV;
```

Comment:
```suggestion
    condition_variable _replicateStart;
```

Reply:
I don't get why we'd remove the type info from the name of the condition variable but not the mutex. They are two coordinated variables for the same purpose.

Reply:
I would've suggested removing Mutex from the other var name too, I just missed it

Reply:
I think this is clearer